recipes/libxshmfence/all/conanfile.py

```diff
@@ -38,31 +40,35 @@ def _settings_build(self):
     def _user_info_build(self):
         return getattr(self, "user_info_build", self.deps_user_info)
 
+    def export_sources(self):
+        export_conandata_patches(self)
```

Comment:
😕 https://github.com/conan-io/conan-center-index/tree/c838687ce150b8c234d93612715935fdc7944e3c/recipes/libxshmfence/all

This does not appear to have patches?

Reply:
Yes, but as the recipe already has the code applying patches, it's better to export them too. One advantage is that if a future version needs some patches, it won't change the recipe ID of existing versions.
If you prefer to remove both the patch export and appliance, I'm OK too.

Reply:
Less is more on this one... it's fine this time since there's no negative impact but people use CCI recipes are a source of truth and this might give the wrong impression.